[2022-12-04T16:12:20.381+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: data_generation.notification scheduled__2022-12-03T04:00:00+00:00 [queued]>
[2022-12-04T16:12:20.388+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: data_generation.notification scheduled__2022-12-03T04:00:00+00:00 [queued]>
[2022-12-04T16:12:20.389+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2022-12-04T16:12:20.390+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2022-12-04T16:12:20.391+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2022-12-04T16:12:20.399+0000] {taskinstance.py:1304} INFO - Executing <Task(BashOperator): notification> on 2022-12-03 04:00:00+00:00
[2022-12-04T16:12:20.407+0000] {standard_task_runner.py:55} INFO - Started process 217 to run task
[2022-12-04T16:12:20.411+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'data_generation', 'notification', 'scheduled__2022-12-03T04:00:00+00:00', '--job-id', '12', '--raw', '--subdir', 'DAGS_FOLDER/generate_data.py', '--cfg-path', '/tmp/tmp7x4i86m8']
[2022-12-04T16:12:20.414+0000] {standard_task_runner.py:83} INFO - Job 12: Subtask notification
[2022-12-04T16:12:20.485+0000] {task_command.py:389} INFO - Running <TaskInstance: data_generation.notification scheduled__2022-12-03T04:00:00+00:00 [running]> on host c7dedbf79ea9
[2022-12-04T16:12:20.542+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=data_generation
AIRFLOW_CTX_TASK_ID=notification
AIRFLOW_CTX_EXECUTION_DATE=2022-12-03T04:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-12-03T04:00:00+00:00
[2022-12-04T16:12:20.545+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2022-12-04T16:12:20.548+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo received data at 2022-12-03']
[2022-12-04T16:12:20.574+0000] {subprocess.py:86} INFO - Output:
[2022-12-04T16:12:20.577+0000] {subprocess.py:93} INFO - received data at 2022-12-03
[2022-12-04T16:12:20.579+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2022-12-04T16:12:20.604+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=data_generation, task_id=notification, execution_date=20221203T040000, start_date=20221204T161220, end_date=20221204T161220
[2022-12-04T16:12:20.664+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2022-12-04T16:12:20.681+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
